### ISSUE TITLE
Update Maven Version!

### ISSUE THREAD

**Issue by noobs2ninjas** [thiagolocatelli/parse4j#75]
I added 1.4 and it does not have the ability to initialize with a self hosted parse URL. I looked at your current pom file and tried to copy the version you have set there and maven cannot find it. Am I missing something or has this not been updated on maven yet? If so. Please update your readme. 

P.S. Great library by the way. Been working on mobile/web with Parse for years. You guys did a great job following their naming conventions and everything!

**Comment by abdielou** [comment]
Agree.... a version release would be great. I think there are way too many breaking changes to even justify a 2.x version.

What I'm currently doing is forking the repo, tagging a version 1.5.x, and using jitpack.io to publish it from my personal repo.

**Comment by noobs2ninjas** [comment]
thats a great Idea! I was just looking at how to fork it and do the same thing. 

Hey, you wouldnt happen to know is it just me or is ParseUser.getQuery() now gone. I just cloned the repo and built a jar file so I could keep working temporarily and I am getting an error even though the readme says its there.

**Comment by abdielou** [comment]
```
        ParseQuery<ParseUser> query = ParseQuery.getQuery("_User")
        user = query.get(id)
```

**Comment by mqueens** [comment]
Getting a weird error that you cat cast ParseObject to ParseUser. Syntax gives me no errors. Weird.

**Comment by noobs2ninjas** [comment]
So I just realized a good 1/2 if not more of the documentation is copied directly from the Android library. Definitely needs to be re-written. So, thats where my confusion on the getQuery() came from. In the android version you can call ParseUser.getQuery() and it returns ParseQuery.getQuery("_User") automatically. That documentation is in the readme as well but not in this library.

**Comment by mortizbey** [comment]
Please update maven version! At least publish this version to the central repository. I think its way due ;)!

**Comment by kapalex** [comment]
@thiagolocatelli joining the request for a new maven version
having to fork it just for the new initialize method really makes little to no sense
thanks!
